FirebaseAuth/Tests/SampleSwift/AuthenticationExample/Models/AuthProvider.swift

```diff
@@ -16,6 +16,7 @@ import UIKit
 
 /// Firebase Auth supported identity providers and other methods of authentication
 enum AuthProvider: String {
+  case settings = "Settings"
```

Comment:
renamed `AuthProvider` to `AuthMenu`

Thanks for the review. Credit for the controller mostly goes to @ncooke3 since I used his quickstart work as a template.

Ultimately, it would be better to convert all the UIKit to SwiftUI though.